### PR TITLE
Add configuration required for cloudbeat logs, metrics

### PR DESCRIPTION
**Pull request by yashtewari**
## What does this PR do?

Required configuration changes to avoid permission errors when Agent-managed filebeat tries to create the `logs-elastic_agent.cloudbeat-*` index on ES. 

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] Configuration files look appropriate.

## How to test this PR locally

Build the `elastic_agent` and `cloud_security_posture` packages, bring the stack up along with the [associated changes on Kibana](https://github.com/elastic/kibana/pull/132452). Running an Agent with the CIS integration added should show the option for cloudbeat logs on the logs page, as shown in the screenshot.

## Screenshots

Screenshot 2022-05-18 at 7 38 08 PM

### CODE REVIEW

File: packages/elastic_agent/data_stream/cloudbeat_logs/fields/ecs.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+- external: ecs
+  name: ecs.version
+- name: log
```
Comment (andrewkroh) [inline packages/elastic_agent/data_stream/cloudbeat_logs/fields/ecs.yml:3]: `log.level` is an ECS field and can be specified more concisely as

```
- external: ecs
  name: log.level
```

File: packages/elastic_agent/data_stream/cloudbeat_logs/fields/fields.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  title: Elastic Agent
+  description: Fields related to the Elastic Agents
+  type: group
+  fields:
+    - name: id
+      type: keyword
+      ignore_above: 1024
+      description: Elastic Agent id.
+    - name: process
+      level: extended
```
Comment (andrewkroh) [inline packages/elastic_agent/data_stream/cloudbeat_logs/fields/fields.yml:14]: `level` and `example` are not attributes that are honored by `package-spec` [^1]. Currently they are allowed by our validation, but they serve no purpose and should not be included.

[^1]: https://github.com/elastic/package-spec/blob/eac675ab2c97f6bd8bdf80cd4c81b370fa4ba421/versions/1/integration/data_stream/fields/fields.spec.yml

Comment (yashtewari) [reply packages/elastic_agent/data_stream/cloudbeat_logs/fields/fields.yml]: I removed those attributes from all the files.

File: packages/elastic_agent/data_stream/cloudbeat_logs/fields/ecs.yml (hunk 1 above)
Comment (yashtewari) [reply packages/elastic_agent/data_stream/cloudbeat_logs/fields/ecs.yml]: Updated, but not too sure if it's correct.

**Review by ph — APPROVED** [review]
This looks good to me, @elastic/elastic-agent-control-plane Please have another eye on this.

**Pull request merged**